projects/addon-commerce/components/input-cvc/input-cvc.component.ts

```diff
@@ -53,17 +53,15 @@ export class TuiInputCVCComponent
     @tuiRequiredSetter()
     set length(length: TuiCodeCVCLength) {
         this.exampleText = '0'.repeat(length);
-        this.textMaskOptions = {
+        this.maskOptions = {
             mask: new Array(length).fill(TUI_DIGIT_REGEXP),
-            guide: false,
         };
     }
 
     exampleText = '000';
 
-    textMaskOptions: TuiTextMaskOptions = {
+    maskOptions: MaskitoOptions = {
```

Comment:
breaking?

Reply:
It doesn't matter. Those should probably be `readonly`, btw.

Reply:
> Those should probably be readonly, btw.

It cannot be readonly because it can be changed in setter `length` (few lines above)